src/config/environments.js

```diff
@@ -0,0 +1,25 @@
+import dotenv from 'dotenv';
```

Comment:
I don't think we need this file, it was replaced with `config.js`. check it out during rebasing.